lib/model.py

```diff
@@ -103,14 +149,14 @@ def set_multi_GPU(self):
 
         # Data parallelism is required to use multi-GPU
         self.G = torch.nn.parallel.DistributedDataParallel(self.G, device_ids=[self.CONFIG['BASE']['GPU_ID']], broadcast_buffers=False, find_unused_parameters=True).module
-        self.D = torch.nn.parallel.DistributedDataParallel(self.D, device_ids=[self.CONFIG['BASE']['GPU_ID']]).module
+        if self.D : self.D = torch.nn.parallel.DistributedDataParallel(self.D, device_ids=[self.CONFIG['BASE']['GPU_ID']]).module
```

Comment:
Note that this line will occur an error if the instance do not have the attribute 'D'.